README.md

```diff
@@ -62,23 +59,34 @@ Additional requirements for Apple:
 * XCode 11+ ([download](https://developer.apple.com/xcode/))
 * XCode command line tools: `xcode-select --install`
 
+
 To open the macOS project on XCode:
 ```
-open ./platforms/ios/Outline.xcodeproj
+open ./platforms/osx/Outline.xcodeproj
 ```
 
 To open the iOS project on XCode:
 ```
-open ./platforms/osx/Outline.xcodeproj
+open ./platforms/ios/Outline.xcodeproj
 ```
 
 To build for macOS (OS X), run:
 
-    npx gulp build --platform=osx
+```
+npx gulp build --platform=osx
+```
+
+To build for iOS, you'll need `ios-deploy` installed globally:
 
-To build for iOS, run:
+```
+npm install -g ios-deploy
```

Comment:
Is this new? We don't like global installations, why can't is be local?
If this is needed, we should mention in the dev requirements above instead.

Reply:
it's something that cordova runs. I can add try it as a local dep but idk if it'll work

Reply:
As far as I can tell this wasn't needed before. What's breaking that you are trying to fix?

Reply:
When I run `npx gulp build --platform=ios` I get:

```
ios-deploy was not found. Please download, build and install version 1.9.2 or greater from https://github.com/ios-control/ios-deploy into your path, or do 'npm install -g ios-deploy'
[10:24:22] 'cordovaCompile' errored after 1.56 s
[10:24:22] Error: exited with error code: 1
    at ChildProcess.onexit (/Users/daniellacosse/outline-client/node_modules/end-of-stream/index.js:40:36)
    at ChildProcess.emit (events.js:314:20)
    at ChildProcess.EventEmitter.emit (domain.js:506:15)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:276:12)
    at Process.callbackTrampoline (internal/async_hooks.js:126:14)
```

However, now that I know the `do` script adds the relevant `node_modules/.bin` to the path, I'm pretty sure just adding it locally will work

Reply:
Yep!

```
** EXPORT SUCCEEDED **

[10:27:03] Finished 'cordovaCompile' after 28 s
[10:27:03] Finished 'build' after 54 s
```